Code/Framework/AzCore/AzCore/Statistics/NamedRunningStatistic.h

```diff
@@ -52,12 +52,12 @@ namespace AZ
 
             static const char * GetCsvHeader()
             {
-                return "Name, Units, numSamples, avg, min, max, stdev";
+                return "Name, Units, numSamples, avg, min, max, stdev\n";
```

Comment:
We would want to write out thread ids or names as well. Without the thread id, if a budget is tracked across multiple threads, we wouldn't be able to tell which numbers are from which thread even though the Budget class logs them separately. We could probably use stat id(which is currently a CRC that the StatisticsManager stores) as a stand-in for thread id.